src/commands/serve.js

```diff
@@ -25,6 +26,24 @@ const { generateSite, createHTMLFile } = require('../utils/generate-site.js');
 const { clearBundleCache } = require('../utils/abell-bundler');
 const { getFirstLine, rmdirRecursiveSync } = require('../utils/abell-fs');
 
+const interfaces = os.networkInterfaces();
+
+/**
+ *
+ * @desc get network address
+ * @return {Number} address
+ */
+function getNetworkAddress() {
+  for (const name of Object.keys(interfaces)) {
+    for (const interface of interfaces[name]) {
+      const { address, family, internal } = interface;
+      if (family === 'IPv4' && !internal) {
+        return address;
+      }
+    }
+  }
+}
```

Comment:
```suggestion
/**
 *
 * @desc get network address
 * @return {Number} address
 */
function getNetworkAddress() {
  const interfaces = os.networkInterfaces();
  
  for (const name of Object.keys(interfaces)) {
    for (const interface of interfaces[name]) {
      const { address, family, internal } = interface;
      if (family === 'IPv4' && !internal) {
        return address;
      }
    }
  }
}
```

I think we can add interfaces inside function since we're not using that variable anywhere else.